web-ui/src/backup_account/page.js

```diff
@@ -36,13 +37,20 @@ export class Page extends React.Component {
 
   saveBackupEmail = (status) => {
     this.setState({ status });
-  }
+  };
 
   mainContent = () => {
     if (this.state.status === 'success') return <Confirmation />;
     return <BackupEmail onSubmit={this.saveBackupEmail} />;
   };
 
+  showSnackbarOnError = (t) => {
+    if (this.state.status === 'error') {
+      return <SnackbarNotification message={t('backup-account.error.submit-error')} isError />;
+    }
+    return undefined;
```

Comment:
yeah, its not ideal but we can probably put a comment next to it to explain why we are doing something silly like this.